test/github-trigger

```diff
@@ -12,16 +12,11 @@ def main():
     parser.add_argument('-f', '--force', action="store_true",
                         help='Force setting the status even if the program logic thinks it shouldn''t be done')
     parser.add_argument('pull', help='The pull request to trigger')
-    parser.add_argument('context', help='The github task context to trigger')
+    parser.add_argument('context', nargs='?', help='The github task context to trigger')
```

Comment:
The help should specify how to trigger all listed contexts (don't pass any contexts). Should we list testinfra.DEFAULT_VERIFY contexts here as an aid?